test/e2e/e2e_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2016 The prometheus-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"github.com/coreos/prometheus-operator/test/e2e/framework"
+	"testing"
+	"time"
+)
+
+func TestCreateCluster(t *testing.T) {
+	f := framework.Global
+	testProm, err := framework.CreatePromCluster(f, framework.MakePromCluster("prometheus-test", 1))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer func() {
+		if err := framework.DeletePromCluster(f, testProm.Name); err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	if _, err := framework.WaitForPodsReady(f.KubeClient, f.Namespace.Name, time.Minute*2, 1, framework.PrometheusListOptions(testProm.Name)); err != nil {
+		t.Fatalf("failed to create 1 Prometheus instances: %v", err)
+	}
```

Comment:
Fine as a basic test in general. We should probably perform at least one check though that we can actually reach the server. Even if it's just hitting the health endpoint ourselves once – we don't actually know whether it was configured properly in the manifest.